gk/main.c

```diff
@@ -649,6 +588,67 @@ alloc_gk_conf(void)
 }
 
 int
+lua_init_grantors(struct gk_config *gk_conf,
```

Comment:
This function and everything that uses it belogs to another patch. It's not used by this patch.